README.md

```diff
@@ -104,19 +104,27 @@ class OrderController
      * @param Order[] $orders
      */
     #[Route('/orders/create', methods: ['POST'], name: 'create_orders')]
-    public function createOrders(Order ...$orders): Ok
+    public function createOrders(Order ...$orders): Response
     {
          // create orders
 
-        return Ok::new();
+        return new Response('Success!', Response::HTTP_CREATED);
     }
 
     #[Route('/order/create', methods: ['POST'], name: 'create_order')]
-    public function createOrder(Order $order): Ok
+    public function createOrder(Order $order): Order
     {
-        // create order
+        try {
+            $this->orderRepository->create($order);
+        } catch (InvalidOrder $e) {
+            // you can also handle exceptions using violines/rest-bundle
+            // add #[Violines\RestBundle\HttpApi\HttpApi]
+            // or @Violines\RestBundle\HttpApi\HttpApi
+            // and configure the serializer to serialize it to your needs
+            return $e;
```

Comment:
this won't work, or? returning an exception?

I'd suggest to leave this part out. I think it is hard to explain the ErrorInterface in short, but I would say, it is not necessary, it is explained in the wiki.